langmeta/langmeta/shared/src/main/scala/org/langmeta/semanticdb/internal/package.scala

```diff
@@ -19,6 +19,23 @@ import org.langmeta.{semanticdb => d}
 package object semanticdb {
   implicit class XtensionSchemaDatabase(sdatabase: s.Database) {
```

Comment:
Now that we're working with `s.Database`, can we remove `scalametaJVM` from the dependencies of `semanticdb-scalac-core`?

Reply:
We should be able to remove scalametaJVM from the dependencies.